### PR TITLE
Update deviceinfo.js, Fix hasNotch() support for LG phones

### PR DESCRIPTION
**Pull request by beaulieufrancois**
Fix hasNotch() support for LG phones 

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| Android |    ✅  |

### CODE REVIEW

File: deviceinfo.js [hunk 1]
```diff
@@ -72,15 +72,15 @@ const devicesWithNotch = [
...
   },
   {
     brand: 'LG',
-    model: 'G7 ThinQ',
+    model: 'LM-G710', //G7 ThinQ
   },
   {
     brand: 'LG',
-    model: 'G7+ ThinQ',
+    model: 'LM-V405', //V40 ThinQ
```
Comment (machour) [inline deviceinfo.js]: Maybe the old values work for other build of those phones.
Can you keep the old one & add new entries instead?

**Comment by machour** [comment]
Merci monsieur!

**Comment by beaulieufrancois** [comment]
Au plaisir!

**Comment by machour** [comment]
Published in 0.26.4 🎉

**Comment by GiladR1979** [comment]
This is still happening on 5.4.0. hasNotch() returns false.
DeviceInfo.getBrand() returns "lge" and not "lg".

**Comment by GiladR1979** [comment]
This is still happening on 5.4.0. 
DeviceInfo.getBrand() returns "lge" and not "lg". => getting "false" in hasNotch() on LG G7 ThinQ

**Comment by mikehardy** [comment]
@GiladR1979 you've mentioned it in 3 places now, which takes the same time as a pull request :-), please make a PR and I'll merge it!

**Comment by GiladR1979** [comment]
Oops, I thought I deleted one response and responded in a wrong place a second time.
Anyway is copying the LG brand to LGE a solution? Its not a pretty one. I don't know why the brand is "LGE" and not "LG" in the first place.

**Comment by mikehardy** [comment]
The entire hasNotch() implementation is a little silly if you think about it. Copy/paste is no better or worse than anything given how it works now. 

What you really want (to know how to display) is handled better by https://github.com/th3rdwave/react-native-safe-area-context

Everyone should really move there but I provide hasNotch here still because it's always been here and doesn't really hurt to maintain as long as the community keeps it working well

**Pull request merged**